applications/DEMApplication/python_scripts/DEM_benchmarks_class.py

```diff
@@ -4249,21 +4243,11 @@ def generate_graph_points(self, modelpart, rigid_face_model_part, cluster_model_
             for node in modelpart.Nodes:
                 if node.Id == 10:           ### stage 0 - simple dem
 
-                    force_node_x = node.GetSolutionStepValue(TOTAL_FORCES)[0]
-                    force_node_y = node.GetSolutionStepValue(TOTAL_FORCES)[1]
-                    force_node_z = node.GetSolutionStepValue(TOTAL_FORCES)[2]
-
-                    angular_node_x = node.GetSolutionStepValue(ANGULAR_VELOCITY)[0]
-                    angular_node_y = node.GetSolutionStepValue(ANGULAR_VELOCITY)[1]
-                    angular_node_z = node.GetSolutionStepValue(ANGULAR_VELOCITY)[2]
-
                     displacement = [0]*3
-                    displacement[0] = node.X - node.X0
-                    displacement[1] = node.Y - node.Y0
-                    displacement[2] = node.Z - node.Z0
+                    displacement[0], displacement[1], displacement[2] = GetDisplacement(node)
 
-                    force_node = sqrt(force_node_x*force_node_x + force_node_y*force_node_y + force_node_z*force_node_z)
-                    angular_node = sqrt(angular_node_x*angular_node_x + angular_node_y*angular_node_y + angular_node_z*angular_node_z)
+                    force_node = sqrt(sum([node.GetSolutionStepValue(TOTAL_FORCES)[i]**2 for i in range(3)]))
+                    angular_node = sqrt(sum([node.GetSolutionStepValue(ANGULAR_VELOCITY)[i]**2 for i in range(3)]))
```

Comment:
This whole piece of code is repeated many times, let us encapsulate it in a function, something like MeasureError().

Reply:
fixed